docs/index.yml

```diff
@@ -0,0 +1,193 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: YnvestTubeApi
+host: "127.0.0.1:8000"
+consumes:
+  - "application/json"
+produces:
+  - "application/json"
+paths:
+  /users:
```

Comment:
This endpoint is useless for the frontend. Guess it could only be used for debuging but i am not sure if it is worth to implement it.